src/components/Extensibility/components-form/MonacoRenderer.js

```diff
@@ -41,18 +41,17 @@ export function MonacoRenderer({
   required,
   resource,
   nestingLevel = 0,
-  originalResource,
 }) {
   const { tFromStoreKeys, t: tExt } = useGetTranslation();
 
   const value = getValue(storeKeys, resource);
   const jsonata = useJsonata({
-    resource: originalResource,
+    resource: resource,
     scope: value,
     value,
   });
 
-  const language = getLanguage(jsonata, schema, value, resource);
+  const language = getLanguage(jsonata, schema);
```

Comment:
good find! :+1: 
no idea why the two extra parameters were there (and it was me who wrote that :laughing:  )